protocol/bots.proto

```diff
@@ -10,23 +10,47 @@ option go_package = "github.com/towns-protocol/towns/core/node/protocol";
 // This session token can be obtained from the AuthenticationService. If the session token is missing or invalid an
 // Err_UNAUTHENTICATED (code=16) is returned.
 service BotRegistryService {
+    rpc Register(RegisterRequest) returns (RegisterResponse);
     rpc RegisterWebhook(RegisterWebhookRequest) returns (RegisterWebhookResponse);
+    // rpc GetRegisteredBotInfo(GetRegisteredBotInfoRequest) returns (GetRegisteredBotInfoResponse);
```

Comment:
I know this is commented out, but is there a way to get unregistered bot info? If not this should just be GetBotInfo, or even just GetInfo, since this is the bot registry service